src/Congo.RazorPages/Services/ProductsService.cs

```diff
@@ -19,89 +21,17 @@ public ProductsService(IHttpClientFactory httpClientFactory)
-                new()
-                {
-                    Id = 1,
-                    Title = "Screwdriver",
-                    Price = 5.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 2,
-                    Title = "AMD Ryzen 7 5900X",
-                    Price = 499.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 3,
-                    Title = "A Rock",
-                    Price = 0.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 4,
-                    Title = "5-Piece Gardening Kit",
-                    Price = 34.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 5,
-                    Title = "Box of Chocolates",
-                    Price = 9.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 6,
-                    Title = "Window Cleaning Solution",
-                    Price = 4.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 7,
-                    Title = "Paper shredder",
-                    Price = 44.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 8,
-                    Title = "Xbox Controller",
-                    Price = 119.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 9,
-                    Title = "LED Light Bulb - 8 pack",
-                    Price = 44.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-                new()
-                {
-                    Id = 10,
-                    Title = "A Dance With Dragons - Hard Cover",
-                    Price = 39.99m,
-                    ImageUrl = "https://via.placeholder.com/150"
-                },
-            };
-        }
-
-        public void PurchaseProduct(int id)
-        {
-            //
+            /* Uses the HttpClientFactory directly for now, will change it to use
+            Refit when it gets implemented. */
+            var client = _httpClientFactory.CreateClient(nameof(Congo));
+            var response = await client.PostAsync($"{_productsUri}/{productId}/purchase", null);
+            var result = await response.Content.ReadFromJsonAsync<OrderConfirmationResponse>();
+            return result.OrderId;
```

Comment:
We want to return the OrderConfirmationResponse in case it later on contains more props